app/Http/Controllers/Admin/PartnerController.php

```diff
@@ -48,18 +53,18 @@ public function store(Request $request)
     /**
      * Display the specified resource.
      *
-     * @param  Voucher  $voucher
+     * @param  Partner  $partner
      * @return Response
      */
-    public function show(Voucher $voucher)
+    public function show(Partner $partner)
```

Comment:
I would remove the show method as it could be called -> but will result a blank page.